Apps/Patient/test/unit/Services/PatientDataServiceTests.cs

```diff
@@ -158,6 +160,75 @@ public async Task CanGetOrganDonorRegistrationData(bool canAccessDataSource)
             }
         }
 
+        [Fact]
+        public async Task CannotGetCancerScreeningData()
+        {
+            CancerScreeningExam expected = new()
+            {
+                EventType = CancerScreeningType.Recall,
+            };
```

Comment:
I think the variable name `expected` is misleading in this particular test.  Since it doesn't need to be accessed again, I'd suggest inlining it when setting up the mock return value.